syntax = "proto3";

message ScoreRequest {
    double ownership = 1;
    double state = 2;
}

message ValueRequest {
    double gain_loss = 1;
}

message Response {
    double ownership = 1;
    double state = 2;
    double gain_loss = 3;
}

message SetTypeRequest {};

message SetInfoRequest {};

service EmotionService {

    rpc SetScore(ScoreRequest) returns (Response);

    rpc SetValue(ValueRequest) returns (Response);

    rpc SetType(SetTypeRequest) returns (Response);

    rpc SetInfo(SetInfoRequest) returns (Response);
}